tools/accuracy_checker/tests/test_segmentation_metrics.py

```diff
@@ -87,7 +87,7 @@ def test_one_class(self):
         dataset = single_class_dataset()
         dispatcher = MetricsExecutor(create_config(self.name), dataset)
         dispatcher.update_metrics_on_batch(range(len(annotations)), annotations, predictions)
-        expected = generate_expected_result([1.0, 0.0], self.name, dataset.labels)
+        expected = generate_expected_result([1.0], self.name, {0: 'dog'})
```

Comment:
I think it would be clearer if you used `dataset.labels[0]` instead of `'dog'` (and similarly in the other cases).

Reply:
yes, thank you, updated